solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureMetadata.java

```diff
@@ -58,14 +59,20 @@ public void getInstanceTest() throws FeatureException
         .getClass().getCanonicalName());
   }
 
-  @Test(expected = FeatureException.class)
-  public void getInvalidInstanceTest() throws FeatureException
+  @Test
+  public void getInvalidInstanceTest()
   {
-    final Map<String,Object> map = new HashMap<String,Object>();
-    map.put(Feature.NAME_KEY, "test");
-    map.put(Feature.CLASS_KEY, "org.apache.solr.ltr.feature.LOLFeature");
-    store.addFeature(map, "testFstore2");
-
+    final FeatureException expectedException = 
+        new FeatureException("Error loading class 'org.apache.solr.ltr.feature.LOLFeature'");
+    try {
+      final Map<String,Object> map = new HashMap<String,Object>();
+      map.put(Feature.NAME_KEY, "test");
+      map.put(Feature.CLASS_KEY, "org.apache.solr.ltr.feature.LOLFeature");
+      store.addFeature(map, "testFstore2");
+      fail("unexpectedly got here instead of catching "+expectedException);
```

Comment:
Should likely say something "getInvalidInstance failed to throw exception:  " + expectedException